pandas/core/generic.py

```diff
@@ -3151,6 +3152,12 @@ def to_csv(
         decimal : str, default '.'
             Character recognized as decimal separator. E.g. use ',' for
             European data.
+        errors : str, default 'strict'
```

Comment:
The possible values should be listed here, like `{'strict', 'ignore', 'replace'}`.

Reply:
The table at https://docs.python.org/3/library/codecs.html#codec-base-classes has other options like xxmlcharrefreplace. Are those valid here?

Reply:
Can you also note that this only applies when pandas is the one opening the file?

We should validate that the user passing `errors` when they provide a filelike object, which we can't pass `errors` to, raises.

Reply:
According to https://docs.python.org/3/library/codecs.html#codec-base-classes, surrogateescape is in 3.1, surrogatepass is in 3.4, namereplace and backslashreplace are in 3.5. So, may I add only `backslashreplace` option? Or should I add all error handling options?

Reply:
Sorry, we can handle all error options because current pandas depends on Python 3.7.

Reply:
We require on Python 3.5.3 or newer.

Reply:
I know we like short kwargs, but I think ``encoding_errors`` would be better here

Reply:
Either is fine by me.